### ISSUE TITLE
Add baritone API support

### ISSUE THREAD

**Issue by ccuser44** [Wurst-Imperium/Wurst7#386]
Baritone has a fabric API and forge API as a mod.

When the API is present wurst should have integration features which become enabled.

Like for example `.b BARITONECOMMAND` would allow the use of baritone commands `#BARITONECOMMAND`

Also you should be able to configure baritone via wurst.

Also add support for commands like baritones automine and autowalk and follow.

And possibly other interesting stuff.

**Comment by ccuser44** [comment]
But what I mean that you shoudlnt have baritone inside wurst. how it should be done is to have it utulise the baritone api mod BUT ONLY IF IT IS PRESENT

**Comment by ccuser44** [comment]
Or it could toallly be integrated idk. But idk you decide

**Comment by Drakesdrs** [comment]
It is already

**Comment by ccuser44** [comment]
Is it?

**Comment by ccuser44** [comment]
> It is already

I don't think so. Ihave not seen any proof of this

**Comment by Soulevoker** [comment]
I haven't seen any code that integrates with Baritone

**Comment by ccuser44** [comment]
> I haven't seen any code that integrates with Baritone

Me neither. I am 100% sure wurst doesn't have baritone API support.

**Comment by Drakesdrs** [comment]
oh I have baritone in mods folder

**Comment by Drakesdrs** [comment]
And if you do #help and it will just leak your uuid for no reason

**Comment by ccuser44** [comment]
> oh I have baritone in mods folder

Yes. Baritone does work with Wurst. But what I am is asking for is Baritone API support. Like if you have the baritone API mod installed Wurst would use it and some extra features would activate. Also this would allow you to configure Baritone trough Wurst much easily than with the commands.

**Comment by ccuser44** [comment]
.

**Comment by ccuser44** [comment]
...

**Comment by ccuser44** [comment]
BUMP this is not stale!

**Comment by ccuser44** [comment]
.

**Comment by ccuser44** [comment]
NOT STALE!

**Comment by ccuser44** [comment]
NOT STALE!

**Comment by ccuser44** [comment]
Bump

**Comment by ccuser44** [comment]
BUMP

**Comment by ccuser44** [comment]
Bump

**Comment by ccuser44** [comment]
Bump

**Comment by ccuser44** [comment]
Bump

**Comment by 2EzMyBoi** [comment]
I agree with BOplaid

**Comment by ccuser44** [comment]
Bump

**Comment by ccuser44** [comment]
Bump

**Comment by BOplaid** [comment]
It's my turn to bump

**Comment by BOplaid** [comment]
Seems like OP has to bump.
@ccuser44
